doc/lua_api.txt

```diff
@@ -1936,6 +1936,11 @@ Call these functions only at load time!
 * `minetest.notify_authentication_modified(name)`
     * Should be called by the authentication handler if privileges changes.
     * To report everybody, set `name=nil`.
+* `minetest.check_password_entry(name, entry, password)`
+    * Returns true if the db entry for a player with name matches given
+    * password, false if not, nil if its formatting is invalid.
+    * Only use it for making it possible to log in via the password from
+    * via protocols like IRC, other uses (like inside the game) are frowned upon.
```

Comment:
As you can read above, its the db entry for a given player.

Just note that it is more than just a "hash" or something, it also contains the salt and a format identifier for future proofness. Only the legacy format solely consists of the hash.

I can give it a fancier name like "authenticator entry" or "database entry generated in order to authenticate players with password based login schemes", will that help?